lib/bst.js

```diff
@@ -58,10 +58,7 @@ class BinarySearchTree {
 
   getHeight() {
     const { root } = this;
-    if (root === null) {
-      return undefined;
-    }
-    return root.getHeight();
+    return root ? root.getHeight() : -1;
```

Comment:
Please rewrite function - remove ternary operators.
Also you can write `!root` instead of `root === null`